### DISCUSSION TITLE
Cloud Functions without the Blaze plan

### DISCUSSION THREAD

**Discussion by shauryaaher** [firebase/functions-samples#1001] [Q&A]
Is it possible to use Cloud Functions without a credit card? It was possible before, and there a lot of issues with the new payment rules in India.

**Comment by ryanmio** [comment]
No, you can't use Cloud Functions without a credit card on the Blaze plan. Google requires billing information for the Blaze plan, which is mandatory for using Cloud Functions. This is especially relevant due to recent changes in payment rules in India.

### Options:

1. **Virtual Credit Card**: Some banks offer virtual credit cards which might bypass the issues you're facing.
   
2. **Friends or Family**: If possible, you can use a credit card from a friend or family member to get past the setup phase.

3. **Other Services**: Consider using other serverless function providers that don't require a credit card.

4. **Local Development**: For development purposes, you can use the Firebase Emulator to run Cloud Functions locally.

5. **Contact Support**: If the payment issues are prohibitive, reach out to Google's support to find if there's a workaround specific to India's payment rules.
